framework/include/interfaces/TaggingInterface.h

```diff
@@ -395,6 +396,14 @@ class TaggingInterface
   /// this data member to avoid constant dynamic heap allocations
   std::vector<Real> _absolute_residuals;
 
+  /// Pointer to the eigen problem
+  const EigenProblem * _eigen_problem;
+  /// Pointer to the postprocessor for the Bx norm
+  const PostprocessorValue * _bx_norm = nullptr;
+  /// Whether a residual tag of the eigen object should be scaled with bx norm
+  /// Note: this corresponds to _re_blocks.
```

Comment:
```suggestion
  /// Note: this has size corresponding to the size of \p _re_blocks.
```

Reply:
This variable is no longer needed in the new implementation.